src/coreclr/vm/prestub.cpp

```diff
@@ -2556,17 +2566,33 @@ static PCODE PreStubWorker_Preemptive(
     // No GC frame is needed here since there should be no OBJECTREFs involved
     // in this call due to UnmanagedCallersOnlyAttribute semantics.
 
-    INSTALL_MANAGED_EXCEPTION_DISPATCHER;
-    INSTALL_UNWIND_AND_CONTINUE_HANDLER;
+    EX_TRY
+    {
+        bool propagateExceptionToNativeCode = IsCallDescrWorkerInternalReturnAddress(pTransitionBlock->m_ReturnAddress);
```

Comment:
`CallDescrWorker` is guaranteed to run in cooperative mode, but `PreStubWorker_Preemptive` is preemptive entrypoint so it should not ever be called by called by CallDescrWorker. Are the changes in `PreStubWorker_Preemptive` reachable?